app.js

```diff
@@ -1,12 +1,14 @@
 const express = require('express')
 const app = express()
 
-const {getAllTopics, noEndpoint} = require('./controllers/api.controller')
+const {getAllTopics, noEndpoint, getAllEndpoints} = require('./controllers/api.controller')
 
 app.use(express.json())
```

Comment:
Remove `app.use(express.json())` until you're actually using it. If you're unsure, a little refresher on what `express.json` does is available in the [NC Notes](https://notes.northcoders.com/courses/js-back-end/express#dealing-with-the-body)